lldb/include/lldb/Utility/Status.h

```diff
@@ -183,6 +186,9 @@ class Status {
   ///     NULL otherwise.
   const char *AsCString(const char *default_error_str = "unknown error") const;
 
+  /// Get the error in machine-readable form.
+  StructuredData::ObjectSP GetErrorData() const;
```

Comment:
We have prior art of calling these methods `GetAsStructuredData`. Let's use that for consistency, unless there's a good reason not to.